### ISSUE TITLE
upgrade version

### ISSUE THREAD

**Issue by moreje** [MRColorR/peer2profit#3]
the current P2P in container version is v0.48 while peer2profit has released 0.65.
Do you consider upgrade your image?
Thnk you

**Comment by MRColorR** [comment]
Could you send me the direct link to the download? 
On their telegram bot, i see only the 0.48, i don't know why.

**Comment by moreje** [comment]
you're right, in telegram...the deb file is 0.48 .... but I have one of my devices which is v0.65...
can't see how to get this
![Capture d'écran 2023-11-05 211449](https://github.com/MRColorR/peer2profit/assets/5374164/39111576-7ee5-4423-9b91-4fc6dfed95f5)

**Comment by MRColorR** [comment]
maybe the support bot could give us a link to download it directly or at least the updated package

**Comment by MRColorR** [comment]
> maybe the support bot could give us a link to download it directly or at least the updated package

just asked them on the support bot  and they gave me the same link pointing to 0.48...so maybe they now update the app internally?

**Comment by jenovauh** [comment]
> you're right, in telegram...the deb file is 0.48 .... but I have one of my devices which is v0.65... can't see how to get this ![Capture d'écran 2023-11-05 211449](https://private-user-images.githubusercontent.com/5374164/280552253-39111576-7ee5-4423-9b91-4fc6dfed95f5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzNDg0NjcsIm5iZiI6MTcxMTM0ODE2NywicGF0aCI6Ii81Mzc0MTY0LzI4MDU1MjI1My0zOTExMTU3Ni03ZWU1LTQ0MjMtOWI5MS00ZmM2ZGZlZDk1ZjUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjVUMDYyOTI3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGY5NTgzZmM3NTA5NzM3YWJhYjViMDc3ZmI0ODBkOTczZmYxNzUwMzcwZGNmNjE3MDg4OGNhOGI3MmEyYjNlYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.J4ULtE7vhG_Eo6IX5TslhBhtRzR6O3NAFwGLPqDg30s)

It is showing 0.65 version LINUX but it is running on Windows machine from what I can tell as my referral is running it on Windows OS.
